lib/axiosHttpClient.ts

```diff
@@ -105,14 +100,6 @@ export class AxiosHttpClient implements HttpClient {
       bodyType === "function" ? httpRequest.body() :
       httpRequest.body;
 
-    const userUploadProgress = httpRequest.onUploadProgress;
```

Comment:
Sections like this are just about supporting browser features. I wasn't sure whether to go through and delete them all. Maybe it's not super important either way?

Reply:
Does this mean we don't support upload and download progress reporting in node.js now?

Reply:
These handlers never had an effect in node.js, which I guess is why I was inclined to remove them now that we only expect to use this module in node.js environments. We have to instead ask our node.js users to wrap their request body stream to get upload progress notifications.